### DISCUSSION TITLE
Отправить файл из панели

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#362] [Как это сделать?] [answered]
Есть возможность отправить на почту файлы, которые сохранились в панели ( в таблице еще нет )?
Использую = :emailSend(to: "vldekov@yandex.ru"; title: "*"; body: "тестовое письмо"; files: #pnl.test_fayl), но выдает ошибку
![image](https://user-images.githubusercontent.com/114401808/235342837-ceb00a7b-af7c-4441-8a99-8545144d590c.png)

**Comment by poluninatm** [comment, marked as answer]
В содержимом pnl.files при подгруженном, но не сохраненном файле содержится что-то такое: 
<img width="475" alt="Снимок экрана 2023-05-10 в 12 36 13" src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/2241864/bcbcf700-33d1-467f-896b-c0fab9a92b20">
`emailSend` ожидает, что файл придет либо уже сохраненный файл с параметром file, либо массив, содержащий filestring. 

Возможно, добавим эту возможность. Сейчас методами Тотума это можно сделать только сохранив файл, например, во временную таблицу и забрав оттуда получившийся параметр `file`
